test/decl/ext/extensions.swift

```diff
@@ -46,6 +46,20 @@ extension S1 {} // no-error
 extension S1.Type {} // expected-error {{cannot extend a metatype 'S1.Type'}}
 extension S1.NestedStruct {} // no-error
 
+struct S1_2 {
```

Comment:
Let me know if you'd rather me remove these since it's already tested elsewhere.